addons/explosives/functions/fnc_isAllowedDefuse.sqf

```diff
@@ -0,0 +1,21 @@
+#include "script_component.hpp"
+/*
+ * Author: Walthzer
+ * Check if Ace is allowed to add a defusal action to a mine.
+ *
+ * Arguments:
+ * 0: Mine <OBJECT>
+ *
+ * Return Value:
+ * Allowed <BOOLEAN>
+ *
+ * Example:
+ * [_mine] call ace_explosives_fnc_isAllowedDefuse
+ *
+ * Public: Yes
+ */
+
+params [["_mine", objNull, [objNull]]];
+TRACE_1("params",_mine);
+
+!(_mine in GVAR(excludedMines));
```

Comment:
```suggestion
!(_mine in GVAR(excludedMines))
```
Returns should not have a semi-colon to indicate that they are returns.

Reply:
Force of habit, listen to much to the linter